### DISCUSSION TITLE
connecting to digital ocean managed mongodb database

### DISCUSSION THREAD

**Discussion by ajay-paramlogic** [vercel/community#585] [Help]
The connection string of mongodb looks like below which is added as an environment variable.

MONGODB_URI=mongodb+srv://username:password@db-name-12345.mongo.ondigitalocean.com/mydatabase?authSource=admin&replicaSet=db-name&tls=true&tlsCAFile=ca-certificate.crt

ca-certificate.crt is required at the root of the project for this to work. How do I get ca-certificate.crt file to the root of the project in Vercel deployment? Or whether the file can be copied anywhere and that location can be referred in the connection string.

**Comment by npapagna** [comment]
I'm facing a similar issue.

@ajay-paramlogic how did you manage to solve it?
